tests/performance-tests/src/utils/dataService.ts

```diff
@@ -0,0 +1,511 @@
+import { sleep } from 'k6';
+import http, { RefinedResponse } from 'k6/http';
+import createClient from './httpClient';
+import TestData from '../tests/testData';
+
+const applicationJsonHeaders = {
+  'Content-Type': 'application/json',
+};
+
+export interface SubjectMeta {
+  filters: {
+    [filter: string]: {
+      options: {
+        [filterGroup: string]: {
+          options: {
+            value: string;
+          }[];
+        };
+      };
+    };
+  };
+  indicators: {
+    [indicatorGroup: string]: {
+      options: {
+        value: string;
+      }[];
+    };
+  };
+  timePeriod: {
+    options: {
+      code: string;
+      year: number;
+    }[];
+  };
+  locations: {
+    [geographicLevel: string]: {
+      options: {
+        id: string;
+      }[];
+    };
+  };
+}
+
+type Topic = {
```

Comment:
Just wondering why you've used `type` here & not `interface`? Typically for objects, I think `interface` is the preferred convention and `type` for things such as unions or intersections